app/templates/account/billing.hbs

```diff
@@ -1,5 +1,5 @@
 <Billing::Account
   @account={{this.account}} 
-  @plans={{this.plans}} 
   @newSubscription={{this.newSubscription}}
+  @updateBillingQueryParams={{action 'updateBillingQueryParams'}}
```

Comment:
You can use `router` service to navigate to new query parameter right from the component.